CIP-0XXX/README.md

```diff
@@ -0,0 +1,551 @@
+---
+CIP: ?
+Title: Cardano Decentralized Message Queue
+Category: ?
+Status: Proposed
+Authors:
+    - Jean-Philippe Raynaud <jp.raynaud@gmail.com>
+    - Arnaud Bailly <arnaud.bailly@iohk.io>
+    - Marcin Szamotulski <marcin.szamotulski@iohk.io>
+    - Armando Santos <armando.santos@iohk.io>
+    - Neil Davies <neil.davies@iohk.io>
+    - Sebastian Nagel <sebastian.nagel@ncoding.at>
+    - ?
+Implementors: 
+    - Cardano Scaling team <https://github.com/cardano-scaling>
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/?
+Created: 2024-08-02
+License: Apache-2.0
+---
+
+# Cardano Decentralized Message Queue
+
+## Questions
+
+## Abstract
+
+We propose to create a decentralized message diffusion protocol leveraging the Cardano network layer. This protocol allows to follow a topic based publish-subscribe pattern to diffuse messages from publisher nodes to subscriber nodes in a decentralized way.  
+
+The messages can be sent and received by nodes running in a non intrusive way side by side to the Cardano node in order to enable inter-nodes communications.
```

Comment:
> non intrusive way side by side to the Cardano node

This is a weird way to hint at the architecture section. I would just say it instead of being mysterious about it.

```suggestion
The messages will be sent and received by mithril nodes. Mithril nodes are implemented using the cardano node network stack and will have to run side by side with full cardano nodes.
```

Reply:
We want to stay generic in the formulation as Mithril is only one of the use cases